pom.xml

```diff
@@ -1,470 +1,502 @@
-      <artifactId>metrics-log4j2</artifactId>
-      <scope>compile</scope>
-    </dependency>
-    <dependency>
-      <groupId>io.dropwizard.metrics</groupId>
-      <artifactId>metrics-jmx</artifactId>
-      <scope>compile</scope>
-    </dependency>
-    <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <scope>test</scope>
-    </dependency>
-  </dependencies>
-  
-  <build>
-    <plugins>
-      <!-- ======================================================= -->
-      <!--     Compilation.                                        -->
-      <!-- ======================================================= -->
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.11.0</version>
-        <configuration>
-          <source>11</source>  <!-- The -source argument for the Java compiler. -->
-          <target>11</target>  <!-- The -target argument for the Java compiler. -->
-          <debug>true</debug>   <!-- Whether to include debugging information.   -->
-          <encoding>UTF-8</encoding> <!-- The -encoding argument for the Java compiler. -->
-          <fork>${fork.javac}</fork> 
-          <maxmem>${javac.maxHeapSize}</maxmem>
-        </configuration>
-      </plugin>
-    </plugins>
 
-  </build>
+    <build>
+        <plugins>
+            <!-- ======================================================= -->
+            <!--     Compilation.                                        -->
+            <!-- ======================================================= -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.11.0</version>
+                <configuration>
+                    <source>11</source>  <!-- The -source argument for the Java compiler. -->
+                    <target>11</target>  <!-- The -target argument for the Java compiler. -->
+                    <debug>true</debug>   <!-- Whether to include debugging information.   -->
+                    <encoding>UTF-8</encoding> <!-- The -encoding argument for the Java compiler. -->
+                    <fork>${fork.javac}</fork>
+                    <maxmem>${javac.maxHeapSize}</maxmem>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.owasp</groupId>
+                <artifactId>dependency-check-maven</artifactId>
+                <version>9.0.9</version>
+                <!-- configuration>
+                    <nvdApiKey>*******-****-****-****-************</nvdApiKey>
+                </configuration -->
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
```

Comment:
This should not be contributed because:
```
[INFO] --- dependency-check-maven:9.0.9:check (default) @ print-lib ---
[INFO] Checking for updates
[WARNING] An NVD API Key was not provided - it is highly recommended to use an NVD API key as the update can take a VERY long time without an API Key
```
and in fact it takes a very long time, as for the other 10.000 of 240.658  takes me something like 5 minutes to download
```suggestion
```